nemo/lightning/pytorch/callbacks/peft.py

```diff
@@ -122,12 +119,14 @@ def freeze_model(self, model: nn.Module) -> None:
         Returns:
             nn.Module: The transformed model with PEFT applied.
         """
+        if isinstance(model, MegatronParallel) and len(model) > 1:
```

Comment:
@cuichenx I was thinking it might be a good idea to also update the type of model?

e.g. `Union[nn.Module, MegatronParallel, torch.nn.parallel.distributed.DistributedDataParallel]` , WDYT?

Reply:
sure, i can do that